python/cudf_kafka/cudf_kafka/_lib/kafka.pyx

```diff
@@ -1,7 +1,8 @@
 # Copyright (c) 2020, NVIDIA CORPORATION.
 
+cimport cpython
```

Comment:
I assume just cimporting this is necessary equivalent to doing a `#include <Python.h>` in a C++ file? I'm wondering why this is here since there are no `cpython.*` calls below.